integration/src/test/scala/ammonite/integration/ErrorTruncationTests.scala

```diff
@@ -18,7 +18,7 @@ object ErrorTruncationTests extends TestSuite{
     val e = fansi.Str(
       intercept[ShelloutException]{ exec(file) }.result.err.string
     ).plainText
-    assert(e == expected)
```

Comment:
Why is this change necessary? The goal of this test suite is to enforce that these outputs are short and do not contain unnecessary stack traces, and this change would invalidate that. If you want to make it less strict on windows, gate your change with a `Util.windowsPlatform` to allow it to pass less-strictly on windows while still being strict on linux